src/components/Error/Error.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { ErrorType } from '../../utils/Enums/ErrorType';
+
+interface Props {
+  error: ErrorType;
+  handleErrorHide: () => void;
+}
+
+export const Error: React.FC<Props>
+  = ({ error, handleErrorHide }) => {
```

Comment:
Fix this code style everywhere
```suggestion
export const Error: React.FC<Props> = ({ 
  error, 
  handleErrorHide,
}) => {
```